free-programming-books.md

```diff
@@ -1817,7 +1816,7 @@ For resources on Angular, Backbone, D3, Dojo, Ember, Express, jQuery, Knockout,
 * [Advanced R Programming](http://adv-r.had.co.nz) - Hadley Wickham
 * [Introduction to Probability and Statistics Using R](http://cran.r-project.org/web/packages/IPSUR/vignettes/IPSUR.pdf) - G. Jay Kerns (PDF)
 * [Learning Statistics with R](http://health.adelaide.edu.au/psychology/ccs/teaching/lsr/) - Daniel Navarro
-* [learnR](https://github.com/renkun-ken/learnR) - (Work in Progress) - Kun Ren
```

Comment:
I think https://github.com/yuanlisky/learnR is more up-to-date. The author committed there as recently as June 2015. @davisonio could you please use this one?